include/swift/Remote/MetadataReader.h

```diff
@@ -756,6 +757,9 @@ class MetadataReader {
       return BuiltOpaque;
     }
     }
+
+    std::cerr << "fatal error: " << "Unhandled MetadataKind in switch." << "\n";
+    std::abort();
```

Comment:
We get linker errors compiling swiftCore.dll if we use `llvm_unreachable`. The errors are `unresolved external llvm_unreachable_internal`. This is because we can't/don't link any LLVM libraries from the stdlib.